mono/mini/aot-compiler.c

```diff
@@ -9087,6 +9113,11 @@ append_mangled_wrapper (GString *s, MonoMethod *method)
 			append_sig = FALSE;
 		} else if (info->subtype == WRAPPER_SUBTYPE_INTERP_LMF)
 			g_string_append_printf (s, "%s", method->name);
+		else if (info->subtype == WRAPPER_SUBTYPE_AOT_INIT) {
+			g_string_append_printf (s, "%s_", m_class_get_image (method->klass)->assembly->aname.name);
+			g_string_append_printf (s, "%d_", info->d.aot_init.subtype);
```

Comment:
This should one call to `g_string_append_printf`?

Reply:
Seems a bit of a micro-optimization, but I'm sure it can't hurt.